public/Aryan Suri/Sidebar/sidebar.js

```diff
@@ -1,6 +1,22 @@
 "use strict";
-window.addEventListener("load", Sidebar);
+window.addEventListener("load", () => {
+    if (Sidebar && !LibreTexts.active.sidebar) {
+        LibreTexts.active.sidebar = true;
+        Sidebar();
+    }
+});
+activateBeeLine();
 async function Sidebar() {
+    readability();
+    function readability() {
+        $('section.mt-content-container p').css("font-size", sessionStorage.getItem("font_size") + "rem");
+        $('section.mt-content-container').css("margin-left", sessionStorage.getItem("page_width") + "px");
+        $('section.mt-content-container').css("margin-right", sessionStorage.getItem("page_width") + "px");
+        $('section.mt-content-container p').css("text-align", sessionStorage.getItem("text_align"));
+        $("#size").val(sessionStorage.getItem("font_size"));
+        $("#slider-page-width").val(sessionStorage.getItem("page_width"));
```

Comment:
These items may not be initialized so you will likely need to provide default values. One option if this is true is to use falsy coalescing `||` so that if the getItem is undefined it instead returns a default value.

Example: `return (sessionStorage.getItem('nonexistent') || 10) +"px";`